src/components/__tests__/ChartToggles.spec.jsx

```diff
@@ -1,22 +1,30 @@
 import configureMockStore from 'redux-mock-store'
-import {
+import ReduxChartToggles, {
```

Comment:
Glad to see you are making progress on fixing the snapshot tests

Reply:
thanks. eventually we'll get it done..